analysis_options.yaml

```diff
@@ -7,8 +7,7 @@ analyzer:
     dead_code: error
     override_on_non_overriding_method: error
   exclude:
-    - "bazel-*"
-    - "**/goldens/**"
+    - "test/goldens/generated_build_script.dart"
```

Comment:
Do we want to exclude this for sure? We do get some analysis today which might be useful to know if we broke things. We could add ignore for file comments to get rid of the hints.

Reply:
Synced offline. Eventually yes I think we will, but for now the `// ignore_for_file:` doesn't seem to work for me.

Once we stop supported the stable SDK we can bump our min version of code_builder and be able to start generating this file without any lints.